samples/xamarin-forms/Cinephile/Cinephile.ViewModels/UpcomingMoviesListViewModel.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using System.Reactive;
+using System.Reactive.Concurrency;
+using System.Reactive.Linq;
+using ReactiveUI;
+using System.Collections.ObjectModel;
+using Cinephile.Core.Models;
+using DynamicData;
+using DynamicData.PLinq;
+using Cinephile.Core.Infrastructure;
+using Splat;
+
+namespace Cinephile.ViewModels
+{
+    public class UpcomingMoviesListViewModel : ViewModelBase
+    {
+        readonly ReadOnlyObservableCollection<UpcomingMoviesCellViewModel> _movies;
+        public ReadOnlyObservableCollection<UpcomingMoviesCellViewModel> Movies => _movies;
+
+        private UpcomingMoviesCellViewModel _selectedItem;
+        public UpcomingMoviesCellViewModel SelectedItem
+        {
+            get { return _selectedItem; }
+            set { this.RaiseAndSetIfChanged(ref _selectedItem, value); }
+        }
+
+        UpcomingMoviesCellViewModel _itemAppearing;
+        public UpcomingMoviesCellViewModel ItemAppearing
+        {
+            get { return _itemAppearing; }
+            set { this.RaiseAndSetIfChanged(ref _itemAppearing, value); }
+        }
+
+        public ReactiveCommand<int, Unit> LoadMovies 
+        {
+            get;
+            set;
+        }
+
+        public ReactiveCommand<Unit, IRoutableViewModel> OpenAboutView
+        {
+            get;
+            set;
```

Comment:
Probably worth removing those two public setters?

Reply:
You are absolutely right! :)